packages/@textlint/markdown-to-ast/test/fixtures/issue-661-tab-indent-codeblock/output.json

```diff
@@ -52,24 +52,263 @@
                             "raw": "Below are the two types of parameters expected:"
                         },
                         {
-                            "type": "CodeBlock",
-                            "lang": null,
-                            "value": "\t* Type `String` called `code`\n\t* Type `boolean` called `accepted`",
+                            "type": "List",
```

Comment:
* Below are the two types of parameters expected:

	* Type `String` called `code`
	* Type `boolean` called `accepted`